plugin/import_cost.vim

```diff
@@ -35,6 +35,8 @@ let s:default_settings = {
   \ 'split_size': 50,
   \ 'split_pos': 'left',
   \ 'disable_async': 0,
+  \ 'disable_auto': 0,
+  \ 'silent': 0,
```

Comment:
I think users that enable the `virtualtext` features would like it to be enabled by default. Wdyt?

Reply:
`auto` was used to decide weather to register the autocommands. So I am guessing what you are saying is that if they have enabled `virtualtext` we will default to auto?

Reply:
I'm talking about the `silent` option